src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/coreperson/Transformer.kt

```diff
@@ -46,3 +71,13 @@ fun OffenderPhoneNumber.toCprPhoneNumber() =
     phoneType = type.code,
     phoneExtension = extension,
   )
+fun OffenderBelief.toCprBelief() = MockCprBelief(
+  nomisBeliefId = beliefId,
+  religion = belief.description,
```

Comment:
I think some of these should be `code` instead?

Reply:
yes - I agree - I saw the comment yesterday that the cpr team made and mentally made a note - but then physically forgot